board/views.py

```diff
@@ -93,21 +96,25 @@ def get(self, req):
 
             # Validating each param.
             board_uuid = UUID(board_uuid, version=4)
-            Board.objects.get(uuid=board_uuid)
             index = int(index)
             amount = int(amount)
         except:
-            return HttpResponse(status=404)
+            return HttpResponse(status=400)
 
-        if (index < 0 or amount < 0):
+        try:
+            Board.objects.get(uuid=board_uuid)
+        except:
```

Comment:
You should be catching the exceptions you are interested in instead of one giant except failsafe